google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
```

Comment:
rolling back to what? having no FLEX_COMPAT in the enum and having flex compat projects be FLEX?

What I'm trying to say is that if you make flex compat project be STD, then the name of the enum is wrong making it extremely difficult for people to talk about the class because they could be talking about 2 completely different things.